ir/instr.cpp

```diff
@@ -1865,4 +1865,121 @@ unique_ptr<Instr> Memcpy::dup(const string &suffix) const {
   return make_unique<Memcpy>(*dst, *src, *bytes, align_dst, align_src, move);
 }
 
+vector<Value*> ExtractElement::operands() const {
+  return { a, idx };
```

Comment:
I would rename a to v, since it's a vector.

Reply:
fixed :)